client/compiled-lang/en.json

```diff
@@ -2575,20 +2575,26 @@
       "value": "Download shapefile"
     }
   ],
+  "methodology.page.downloadPacket.button3.text": [
+    {
+      "type": 0,
+      "value": "Download technical support document"
```

Comment:
Is this not in i18n'd?

Reply:
It should be. I ran all the scripts and go no errors. Can you point me to where you think it should be to help me out?

Reply:
I meant, the button text will be translated if/when we have the `es.json`?

Reply:
Yup. EVERYTHING should be in the translation framework.